src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -288,6 +293,38 @@ public void lock() throws SmartcarException {
     this.call("security", "POST", body);
   }
 
+
+  /**
+   * Send request to the /engine/oil endpoint
```

Comment:
Update this JavaDoc. It seems to be copy/pasted from the oil endpoint.

Reply:
oopsie